06_modules_lab/02.py

```diff
@@ -1,4 +1,10 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-"""
-
+import sys
+import os
+
+if len(sys.argv) != 3 :
+    print "Not enogh paraeters "

```

Comment:
Better to write the error message inside `sys.exit()`. It prints the error to stderr AND returns an error code to the operating system.